webpack.config.babel.js

```diff
@@ -48,7 +49,11 @@ const baseStyleLoaders = (modules=true) => [
     options: {
       sourceMap: true,
       modules,
-      localIdentName: 'onfido-sdk-ui-[folder]-[local]'
+      getLocalIdent: (context, localIdentName, localName) => {
```

Comment:
I know that customisation is not part of the contract but this might be a breaking change for some of our clients who override styles. Maybe this change should make the next release major. What do you think @rfreitas?